docs/resources/logs_integration_pipeline.md

```diff
@@ -1,27 +1,36 @@
 ---
-page_title: "datadog_logs_integration_pipeline"
+page_title: "datadog_logs_integration_pipeline Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a Datadog Logs Pipeline API resource to manage the integrations.
+  Integration pipelines are the pipelines that are automatically installed for your organization when sending the logs with specific sources. You don't need to maintain or update these types of pipelines. Keeping them as resources, however, allows you to manage the order of your pipelines by referencing them in your datadoglogspipelineorder resource. If you don't need the pipelineorder feature, this resource declaration can be omitted.
```

Comment:
We should maybe open an issue for the main description here doing a prerendering of the markdown. Or does it even appear on the final docs this part ??

Reply:
i'm not seeing this rendered on the registry site, though I could be missing it. 
As far as I can tell the `---` before and after this section are `horizontal rules` in html to break up sections.